python/adelphi/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py27,py36
+
+[testenv]
+deps = pytest
+       docker ~= 4.4.3
+       subprocess32 ~= 3.5.4
+setenv =
+       KEEP_LOGS=1
+       CASSANDRA_VERSIONS=4.0-beta3
```

Comment:
any reason not to include all versions here by default?

Reply:
Nah, this is a holdover from local testing... you're absolutely right, I just need to take this out

Reply:
I moved those settings over to a tox.ini.dev and removed them from the base tox.ini.